src/utils/notifications.ts

```diff
@@ -0,0 +1,131 @@
+import { NotifyClient } from '@walletconnect/notify-client'
+import { getFirebaseToken } from './firebase'
+import { useState, useEffect } from 'react'
+import { getDbEchoRegistrations, getDbSymkeyStore } from './idb'
+
+const ECHO_URL = 'https://echo.walletconnect.com'
+
+const setupSubscriptionSymkey = async (topic: string, symkey: string) => {
+  const [, putSymkey] = await getDbSymkeyStore()
+
+  await putSymkey(topic, symkey)
+}
+
+export const setupSubscriptionsSymkeys = async (topicSymkeyEntries: [string, string][]) => {
+  topicSymkeyEntries.forEach(([topic, symkey]) => setupSubscriptionSymkey(topic, symkey))
+
+  for (const [topic, symkey] of topicSymkeyEntries) {
```

Comment:
Looks like this is iterating `topicSymkeyEntries` twice unnecessarily? Once as `.forEach` above and once as `for...of` loop